Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md

```diff
@@ -40,6 +40,22 @@ The `release` channel is the most recent release of Rook that is considered stab
 The example install assumes you have first installed the [Rook Operator Helm Chart](operator-chart.md)
 and created your customized values.yaml.
 
+**Note:**
```

Comment:
Using a tip will format nicely with an icon
```suggestion
!!! tip
   Instead of copying the entire default `values.yaml`, create a new `values.yaml` file that only includes the settings you want to override.

```